### PR TITLE
Adds script to test connection with other services

### PR DESCRIPTION
**Pull request by tallysmartins**
- The wait-for.sh script was copied from https://github.com/Eficode/wait-for
respecting the MIT LICENSE specified in https://github.com/eficode/wait-for/blob/master/LICENSE.
A copy from the license can be found in the wait-for.sh script file.

usage:
-  `./wait-for.sh host:port  -- command`
-  `./wait-for.sh www.google.com:80  -- echo "Success!"`
-  `./wait-for.sh localhost:8331  --timeout=2 -- echo "Success!"`

### CODE REVIEW

**Comment by AndrewDryga** [comment]
There is a much better/cleaner way to wait for something to be ready. First one is `dockerize` which is external dependency and second one is using `nc`. Here is an example for PostgreSQL:
```
for i in `seq 1 30`;
do
  nc -z localhost 5432 && echo Success && exit 0
  echo -n .
  sleep 1
done
echo Failed waiting for Postgres && exit 1
```.

**Comment by tallysmartins** [comment]
Thanks for your review @AndrewDryga!

Well, this is basically what the `wait-for.sh` script does, so you think we can just add the `port`and `timeout` as input params to your example and have a better/cleaner script?

Another question here that just came into my mind, do you think if the test fail should we abort the benchmark and give a good message to the user or should we try to proceed and see what happens? I mean, do you prefer to exit 0 or exit 1 when the test fail?

**Comment by AndrewDryga** [comment]
@tallysmartins sorry, initially did not notice that you actually use the `nc` in the script. It's purely my bad and everything is fine 👍.

File: Dockerfile [hunk 1]
```diff
@@ -57,6 +57,11 @@ RUN cd ${PROJECT_PATH}
 # Export benchmarks results as a volume
 VOLUME ${BENCHMARKS_OUTPUT_PATH}
 
+# Add script to test TCP connections of external services that are specified
+# as dependencies for the runner container
+COPY wait-for.sh /usr/local/bin
+RUN ln -s usr/local/bin/wait-for.sh / # backwards compat
```
Comment (PragTob) [inline Dockerfile:63]: what is the backwards compatible part here? I know it's somewhere else, I just don't know docker and wonder

Comment (PragTob) [inline Dockerfile:63]: What are we waiting for aka which service? I mean we need a way to specify services or ports to wait for right or am I missing something?

**Review by PragTob — COMMENTED** [review]
Mostly questions from me :)

File: Dockerfile (hunk 1 above)
Comment (PragTob) [reply Dockerfile:63]: excuse this of course just links that makes everything clear, I'm too tired to be reviewing stuff I think now :)

**Pull request merged**